modules/ckanext-ytp_main/ckanext/ytp/i18n/ckanext-ytp_main.pot

```diff
@@ -1615,6 +1615,10 @@ msgid ""
 "resource downloads, comments and translations into consideration."
 msgstr ""
 
+#: ckanext/ytp/templates/package/snippets/dataset_info.html
```

Comment:
These .pot files should be generated. (http://docs.ckan.org/en/latest/extensions/translating-extensions.html)